mkosi/__init__.py

```diff
@@ -7877,28 +7878,28 @@ def expand_specifier(s: str) -> str:
 
 
 def needs_build(args: Union[argparse.Namespace, MkosiArgs]) -> bool:
-    return args.verb == "build" or (args.verb in MKOSI_COMMANDS_NEED_BUILD and (not args.output.exists() or args.force > 0))
+    return args.verb == Verb.build or (args.verb in MKOSI_COMMANDS_NEED_BUILD and (not args.output.exists() or args.force > 0))
 
 
 def run_verb(raw: argparse.Namespace) -> None:
     args = load_args(raw)
 
     prepend_to_environ_path(args.extra_search_paths)
 
-    if args.verb == "genkey":
+    if args.verb == Verb.genkey:
         generate_secure_boot_key(args)
 
-    if args.verb == "bump":
+    if args.verb == Verb.bump:
         bump_image_version(args)
 
-    if args.verb in MKOSI_COMMANDS_SUDO:
+    if args.verb.name in MKOSI_COMMANDS_SUDO:
```

Comment:
Quick hint that you should go the other way there, don't check the string name of the enum is in the list. Instead, change the list to contain enum values instead of strings

Reply:
Hm, that makes sense, if we're getting rid of string comparisons, checking if the string name of the enum is in the list is contradicting exactly what we're trying to do.